### PR TITLE
added API_VIP_CONNECTIVITY_CHECK_IMAGE env var

### PR DESCRIPTION
**Pull request by danielerez**
Added API_VIP_CONNECTIVITY_CHECK_IMAGE env var to skipper.yaml and configmap.

### CODE REVIEW

**Comment by danielerez** [comment]
@tsorya @yevgeny-shnaidman

**Comment by eranco74** [comment]
@danielerez note that if this ENV override is already supported in assisted-service (seems to be) there is no need adding it here.

**Comment by danielerez** [comment]
> @danielerez note that if this ENV override is already supported in assisted-service (seems to be) there is no need adding it here.

also for deploying 

> @danielerez note that if this ENV override is already supported in assisted-service (seems to be) there is no need adding it here.

also if deploying the service using test-infra? cause it doesn't seem to propagate without this.

File: skipper.yaml [hunk 1]
```diff
@@ -46,6 +46,7 @@ env:
   NAMESPACE: $NAMESPACE
   IMAGE_BUILDER: $IMAGE_BUILDER
   CONNECTIVITY_CHECK_IMAGE: $CONNECTIVITY_CHECK_IMAGE
+  API_VIP_CONNECTIVITY_CHECK_IMAGE: $API_VIP_CONNECTIVITY_CHECK_IMAGE
```
Comment (tsorya) [inline skipper.yaml:49]: it is enough to set it here, no need for change in update_assisted_service_cm.py

Comment (danielerez) [reply skipper.yaml:49]: ok, removed from cm.

**Pull request merged**